src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -1261,16 +1274,9 @@ private static void useOverTimeJournal(StringBuilder sql, TenantPgPool pool,
       .append(" ) t").append(i).append(" ON t").append(i).append(".titleEntryId = ")
         .append(titleEntriesTable(pool)).append(".id");
     }
-    sql.append(" WHERE agreementId = $1");
-  }
-
-  Future<Void> getUseOverTimeBook(Vertx vertx, RoutingContext ctx) {
-    // Work in Progress (WIP), hardcoded example JSON
-    ctx.response().setStatusCode(200);
-    ctx.response().putHeader("Content-Type", "application/json");
-    ctx.response().end(ResourceUtil.load("/openapi/examples/report.json"));  // FIXME
-
-    return Future.succeededFuture();
+    sql.append(" WHERE agreementId = $1 AND ")
+        .append(journal ? "NOT " : "")
+        .append("(printISSN IS NULL AND onlineISSN IS NULL)");
```

Comment:
Some question as above.

Reply:
Generated SQL

- for book: `AND NOT (printISSN IS NULL AND onlineISSN IS NULL)`
- for journal: `AND (printISSN IS NULL AND onlineISSN IS NULL)`

This is correct and tested by unit tests.

Reply:
Ah, yes, I see. My bad.